client/src/store/new-request-module.ts

```diff
@@ -2370,32 +2365,43 @@ export class NewRequestModule extends VuexModule {
       } else {
         await errorModule.setAppError({ id: 'complete-payment-error', error: error.message } as ErrorI)
       }
+      return null
     }
   }
   @Action
-  async rollbackNameRequest ({ nrId, action }): Promise<any> {
+  async rollbackNameRequest ({ nrId, action }): Promise<boolean> {
```

Comment:
So I tried to pass nrId and action as simple format parameters but they arrived here undefined!?!? Apparently, because this is a store action, I have to pass in an object.

Dunno, haven't quite figured this out (after all, I'd like to add types), but this works for now.